client/gutenberg/extensions/presets/jetpack/utils/simple-input.js

```diff
@@ -12,7 +12,12 @@ const simpleInput = ( type, props, placeholder, view, onChange ) => {
 		>
 			{ ! isSelected && value !== '' && view( props ) }
 			{ ( isSelected || value === '' ) && (
-				<PlainText value={ value } placeholder={ placeholder } onChange={ onChange } />
+				<PlainText
+					value={ value }
+					placeholder={ placeholder }
+					aria-label={ placeholder }
```

Comment:
This was meant to be `ariaLabel`?

Reply:
Ohhh, I see you're using `simpleInput` only for some things and otherwise directly `PlainText`. 👍 

I'd maybe rename `placeholder` attribute to `label` in that case for added clarity but not a biggie.